types/node-notifier/notifiers/notifysend.d.ts

```diff
@@ -10,10 +10,12 @@ declare namespace NotifySend {
         title?: string;
         message?: string;
         icon?: string;
+        /** Shorthand for timeout 5 seconds if true. Timeout option overrides wait */
+        wait?: boolean;
         /** Specifies the urgency level  (low,  normal,  critical). */
         urgency?: string;
-        /** Specifies  the  timeout  in  milliseconds at which to expire the notification */
-        time?: number;
```

Comment:
IMO this should be left. I could be wrong, but the comments reads:
https://github.com/mikaelbr/node-notifier#usage-notifysend
```md
 Alias for expire-time, time etc
```
the actual code is using `time`:
https://github.com/mikaelbr/node-notifier/blob/ee7916a60ca3d8fc9e5994cdd301c51f5bd56d9a/test/notify-send.js#L104
Otherwise OK

Reply:
These are remnants from the past. I think it is safe to simplify the API to use only one of the aliases, and its better to use timeout as this is the same as for the other notifiers. So this is a more unification of the APIs. Yes, it breaks the API, but node-notifier is bumped as a major also.

Reply:
Somehow I've missed the answer here, sry for the delay!